detox/src/DetoxWorker.js

```diff
@@ -5,7 +5,7 @@ const Client = require('./client/Client');
 const environmentFactory = require('./environmentFactory');
 const { DetoxRuntimeErrorComposer } = require('./errors');
 const { InvocationManager } = require('./invoke');
-const symbols = require('./symbols');
+const symbols = require('./realms/symbols');
```

Comment:
Since "symbols" module is actually very sensitive to the realms (primary / secondary / sandbox ), I thought it is a good idea to keep it together.